ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -1,10 +1,10 @@
 base_images:
   ansible:
-    name: "4.15"
+    name: "4.16"
```

Comment:
I think those are for the installation purpose, so it should follow the start version, which is 4.15 here.

Reply:
good catch.